tests/php/src/Cli/Export/ExportOptions.php

```diff
@@ -0,0 +1,359 @@
+		'/^uagb-version$/',
+		'/^uninstall_plugins$/',
+		'/^upload_(path|space_check_disabled|url_path)$/',
+		'/^uploads_use_yearmonth_folders$/',
+		'/^users_can_register$/',
+		'/^widget_.*$/',
+		'/^woocommerce_(admin_notices|allow_bulk_remove_personal_data|allow_tracking|allowed_countries|anonymize_completed_orders|api_enabled|calc_discounts_sequentially|calc_taxes|cart_redirect_after_add|checkout_pay_endpoint|delete_inactive_accounts|demo_store|downloads_grant_access_after_payment|downloads_require_login|file_download_method|meta_box_errors|queue_flush_rewrite_rules|shipping_debug_mode|show_marketplace_suggestions)$/',
+		'/^woocommerce_(db_)?version$/',
+		'/^woocommerce_email_.*$/',
+		'/^woocommerce_erasure_request_.*$/',
+		'/^woocommerce_myaccount_.*$/',
+		'/^woocommerce_notify_.*$/',
+		'/^woocommerce_ship_to_.*$/',
+		'/^woocommerce_specific_.*$/',
+		'/^woocommerce_stock_(email_recipient|format)$/',
+		'/^woocommerce_trash_.*$/',
+		'/^wp_astra_theme_db_migration_.*$/',
+		'/^wp_user_roles$/',
+		'/^wpforms_(activated|version|version_lite)$/',
+		'/^wpseo_(flush_rewrite|ryte)$/',
+		'/^wpsupercache_.*$/',
+	];
+
+	/**
+	 * Associative array of options and their default values for skipping.
+	 *
+	 * @var array
+	 */
+	const OPTION_DEFAULTS = [
+		'use_balanceTags'                 => '0',
+		'use_smilies'                     => '1',
+		'posts_per_rss'                   => '10',
+		'rss_use_excerpt'                 => '0',
+		'posts_per_page'                  => '10',
+		'date_format'                     => 'F j, Y',
+		'time_format'                     => 'g:i a',
+		'category_base'                   => '',
+		'template'                        => 'astra',
+		'stylesheet'                      => 'astra',
+		'use_trackback'                   => '0',
+		'thumbnail_size_w'                => '150',
+		'thumbnail_size_h'                => '150',
+		'thumbnail_crop'                  => '1',
+		'medium_size_w'                   => '300',
+		'medium_size_h'                   => '300',
+		'large_size_w'                    => '1024',
+		'large_size_h'                    => '1024',
+		'thread_comments'                 => '1',
+		'thread_comments_depth'           => '5',
+		'comments_per_page'               => '50',
+		'medium_large_size_w'             => '768',
+		'medium_large_size_h'             => '0',
+		'wp_page_for_privacy_policy'      => '0',
+		'disallowed_keys'                 => '',
+		'auto_plugin_theme_update_emails' => [],
+		'__uagb_do_redirect'              => '',
+	];
+
+	/**
+	 * Process the export step.
+	 *
+	 * @param ExportResult $export_result Export result to adapt.
+	 *
+	 * @return ExportResult Adapted export result.
+	 */
+	public function process( ExportResult $export_result ) {
+		$option_keys = array_filter(
+			$this->get_option_keys(),
+			[ $this, 'skip_excluded_options' ]
+		);
+
+		$options = $this->fetch_options( $option_keys );
+		$options = array_merge( $options, $this->fetch_theme_mods() );
```

Comment:
Why are the options being merged with the theme mods? How will these be properly re-imported as theme mods? Themes can have their own custom theme mods that aren't among the fixed set that core supports (e.g. colors). To properly handle theme mods, I think they should be exported separately and then have a separate importer that will copy them as-is to the theme mods, with special handling for the select few that need additional processing (e.g. custom logo).

Reply:
That was what the starter sites code from Astra was using which I used as initial test environment. But I agree, it makes more sense to split this out.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/5162/commits/2cd9e326dc2649b8317e27c65f0827aaa992a420